Makefile

```
@@ -171,6 +177,10 @@ image-push: ## Push image with the manager.
 .PHONY: image
 image: image-build image-push ## Build and push image with the manager.
 
+.PHONY: debug-image-build
```

```suggestion
.PHONY: image-build-debug
```
For naming consistency.

---

No. For naming consistency entity changes, `image` => `debug-image`. So `image-build` => `debug-image-build`